src/mrinufft/operators/base.py

```diff
@@ -103,11 +104,11 @@ class or instance of class if args or kwargs are given.
     if args or kwargs:
         operator = operator(*args, **kwargs)
 
-    if autograd:
-        if isinstance(operator, FourierOperatorBase):
-            operator = operator.make_autograd(variable=autograd)
-        else:  # partial
-            operator = partial(operator.with_autograd, variable=autograd)
+    # if autograd:
+    if isinstance(operator, FourierOperatorBase):
+        operator = operator.make_autograd(wrt_data, wrt_traj)
+    else:  # partial
+        operator = partial(operator.with_autograd, wrt_data, wrt_traj)
```

Comment:
```suggestion
    elif (wrt_data or wrt_traj):  # partial
        operator = partial(operator.with_autograd, wrt_data, wrt_traj)
```


With this, we enable the `make_autograd` pipeline only if needed. This way you wont need `torch` always